universal-login-react/src/ui/BackupCodes/BackupCodes.tsx

```diff
@@ -28,7 +27,7 @@ export const BackupCodes = ({deployedWallet, basePath = '', className}: BackupPr
   const generateBackupCodes = async () => {
     try {
       history.replace(join(basePath, 'waitingForBackupCodes'));
-      const {waitToBeSuccess, waitForTransactionHash} = await deployedWallet.generateBackupCodes({...transactionDetails, ...gasParameters});
+      const {waitToBeSuccess, waitForTransactionHash} = await deployedWallet.generateBackupCodes(gasParameters!);
```

Comment:
I think we should use ensureNotNull before using "!"

Reply:
👍